### PR TITLE
Remove unnecessary commas

### PR DESCRIPTION
**Pull request by kafoster11**
I removed unnecessary commas to improve overall readability and clarity. Also fixed a handful of other grammatical errors.

### CODE REVIEW

File: Reference/Templating/Masterpages/index.md [hunk 1]
```diff
@@ -44,22 +44,22 @@ This will output an `<img/>` tag when the template renders:
...
 	<umbraco:macro alias="topnavigation" className="greenList" runat="server" />
 
 [See the full umbraco:macro reference](umbracomacro.md)    
 
 ## Template inheritance
-Templates can inherit other templates and uses 2 elements to merge them. `<asp:contentplaceholder>` and `<asp:content>` To connect one template with another, use the dropdownlist in Umbraco to specify the master template, this will change the template declaration and make a database change.
+Templates can inherit other templates and uses 2 elements to merge them. `<asp:contentplaceholder>` and `<asp:content>` To connect one template with another, use the dropdownlist in Umbraco to specify the master template. This will change the template declaration and make a database change.
 
-Lets imagine we have define the below template structure.
+Let's imagine we have define the below template structure.
```
Comment (dampee) [inline Reference/Templating/Masterpages/index.md]: While we are updating this line: "we have defined" with a d at the end. Right?

**Comment by kafoster11** [comment]
Ah, good catch. Fixed!

**Comment by dampee** [comment]
Hi @kafoster11 
Do you have a profile on OUR.UMBRACO.ORG?  If so, can you send us a link?
Kind regards

**Comment by kafoster11** [comment]
Hi @dampee,
Here is my OUR.UMBRACO.ORG profile.

https://our.umbraco.com/member/275678

I am a brand new member, but look forward to becoming more active.

Cheers!

**Comment by nul800sebastiaan** [comment]
Hi there @kafoster11,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**